nextcloud_update.sh

```diff
@@ -106,8 +106,16 @@ Holding PHP to avoid upgrading to a newer version without migration..."
     apt-mark hold php*
 fi
 
+# Hold MariaDB
+if dpkg -l | grep -q mariadb-common
+then
+    print_text_in_color "$ICyan" "MariaDB installed. \
+    Holding MariaDB to avoid crashes..."
+    apt-mark hold mariadb-common
+fi
+
 # Don't allow MySQL/MariaDB
-if ! grep -q pgsql /var/www/nextcloud/config/config.php || dpkg -l | grep -q mariadb-common
+if ! grep -q pgsql /var/www/nextcloud/config/config.php
```

Comment:
would this solve both problems?

Reply:
This doesn't cover SQLite, MySQL or any other DB as it only checks for psql - I think.

If psql is not in config.php AND mariadb exists, THEN = true.

Reply:
Hm...